Library/Homebrew/tap.rb

```diff
@@ -382,7 +382,7 @@ def contents
   # an array of all {Formula} files of this {Tap}.
   def formula_files
     @formula_files ||= if formula_dir.directory?
-      formula_dir.children.select(&method(:formula_file?))
+      formula_dir.children.select { |file| file.extname == ".rb" }
```

Comment:
`formula_file?` is no longer used after this change. It'd be nicer to have a `formula_file?` (or, probably more accurately, `ruby_file?`) method which is called in the same way as previously and shared by both cases.

Reply:
Initially I removed both `formula_file?` and `cask_file?`, but appareantly they are used in `dev-cmd/update-report.rb` for `brew update`. I did add a `ruby_file?` method and used them in `formula_file?` and `cask_file?`.